lib/shared/widgets/custom_navigation_drawer.dart

```diff
@@ -20,6 +20,9 @@ class CustomNavigationDrawer extends StatelessWidget {
     final size = MediaQuery.of(context).size;
     final locals = AppLocalizations.of(context);
     final auth = context.read<AuthCubit>().state;
+    final showFamilyItem = auth.user.country == 'US' &&
+        !auth.user.needRegistration &&
+        auth.user.mandateSigned;
```

Comment:
Could you follow the same way I did it throughout the app?
`auth.user.country == Country.us.countryCode`

Reply:
fixed, thanks